playbooks/set_scale.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Ensure cluster machinesets with given instance type have given scale (total number of replicas equals requested number of replicas)
```

Comment:
`Ensures that the cluster has the request number of machines of a given machine-type` ?

Reply:
I prefer to use [instance type](https://aws.amazon.com/ec2/instance-types/) rather than machine type, it's also the name of the variable (`machineset_instance_type`)

Reply:
sure, `instance-type` is the correct one, machine-type was a typo :)
my suggestion was more for having a shorter description